src/accounts/BitcoinAccount.js

```diff
@@ -91,19 +93,19 @@ export default class BitcoinAccount implements IAccount<AccountData> {
     }
 
     // set id
-    if (!_accountData.id) {
-      accountData.xpub
-        ? (_accountData.id = JSON.stringify({
-            cryptoType: accountData.cryptoType,
-            walletType: accountData.walletType,
-            xpub: accountData.xpub
-          }))
-        : (_accountData.id = JSON.stringify({
-            cryptoType: accountData.cryptoType,
-            walletType: accountData.walletType,
-            address: accountData.address
-          }))
-    }
+    _accountData.hdWalletVariables.xpub
```

Comment:
should probably also make changes to EthereumAccount.js?

Reply:
already made